lib/rules/metadata/profile.js

```diff
@@ -145,6 +144,14 @@ exports.check = async function (sr, done) {
             sr.throw(
                 `[EXCEPTION] The document "${docTitle}" seems to be an Editor's Draft, which is not supported.`
             );
+        } else if (
+            sr.jsDocument.querySelector(
+                'link[href^="https://www.w3.org/StyleSheets/TR/"]'
+            )
+        ) {
+            sr.throw(
+                `Pubrules is having a hard time identifying the profile of the document "${docTitle}" from its &lt;h2&gt;. Please make sure the &lt;h2&gt; is in a valid format: <pre><code>&lt;h2&gt;W3C @@type of the document@@ DD Month YYYY&lt;/h2&gt;</code></pre> Recommendations:<ul><li>Document for <a href="./doc/rules/?profile=FPWD#dateTitleH2">First Public Working Draft</a></li><li>Document for <a href="./doc/rules/?profile=WD#dateTitleH2">Working Draft</a></li><li>Document for <a href="./doc/rules/?profile=CR#dateTitleH2">Candidate Recommendation Snapshot</a></li><li>Document for <a href="./doc/rules/?profile=CRD#dateTitleH2">Candidate Recommendation Draft</a></li><li>Document for <a href="./doc/rules/?profile=CR-AMENDED#dateTitleH2">Candidate Amended Recommendation</a></li><li>Document for <a href="./doc/rules/?profile=PR#dateTitleH2">Proposed Recommendation</a></li><li>Document for <a href="./doc/rules/?profile=PR-AMENDED#dateTitleH2">Proposed Amended Recommendation</a></li><li>Document for <a href="./doc/rules/?profile=REC#dateTitleH2">Recommendation</a></li><li>Document for <a href="./doc/rules/?profile=REC-AMENDED#dateTitleH2">Amended Recommendation</a></li><li>Document for <a href="./doc/rules/?profile=REC-RSCND#dateTitleH2">Rescinded Recommendation</a></li></ul>Notes:<ul><li>Document for <a href="./doc/rules/?profile=FPIG-NOTE#dateTitleH2">First Public Interest Group Note</a></li><li>Document for <a href="./doc/rules/?profile=IG-NOTE#dateTitleH2">Interest Group Note</a></li><li>Document for <a href="./doc/rules/?profile=FPWG-NOTE#dateTitleH2">First Public Working Group Note</a></li><li>Document for <a href="./doc/rules/?profile=WG-NOTE#dateTitleH2">Working Group Note</a></li></ul>Submissions:<ul><li>Document for <a href="./doc/rules/?profile=MEM-SUBM#dateTitleH2">Member Submission</a></li></ul>`
```

Comment:
It might be better to generate the list from the profiles like we do in the [doc page](https://github.com/w3c/specberus/blob/main/views/doc/index.handlebars) instead of hardcoding the list. It's easy to forget if we add/remove profiles.